usr/lib/pkcs11/common/mech_dsa.c

```diff
@@ -52,7 +52,7 @@ dsa_sign( STDLL_TokData_t     * tokdata,
       if (rc == CKR_OBJECT_HANDLE_INVALID)
           return CKR_KEY_HANDLE_INVALID;
       else
-        return rc
+        return rc;
```

Comment:
Apparently this file was not being compiled or used by any token up to now, as the compilation was not breaking on it